core/App/localization/en/index.ts

```diff
@@ -36,6 +37,7 @@ const translation = {
   "Error": {
     "Unknown": "Unknown Error",
     "Problem": "A problem has occurred",
+    "ErrorCode": "Error code",
```

Comment:
To include in the fr localization file: "Code d'erreur"

Reply:
Added